vllm/config.py

```diff
@@ -1515,6 +1515,11 @@ class LoadConfig:
     use_tqdm_on_load: bool = True
     """Whether to enable tqdm for showing progress bar when loading model
     weights."""
+    pt_load_map_location: str = "cpu"
```

Comment:
Shouldn't map_location be a map? Is only a string expressive enough to tell the target device if we have multiple GPUs?

Reply:
I see, from https://pytorch.org/docs/stable/generated/torch.load.html seems like it could be quite complicated, it can even be a Callable: `map_location (Optional[Union[Callable[[Storage, str], Storage], device, str, Dict[str, str]]]) – a function, torch.device, string or a dict specifying how to remap storage locations`

We can definitely add dict if it's needed, for now seems a string is enough. (default is hardcoded to `cpu` for all cases)

Reply:
I feel right now it's cutting the corner. If we do "cuda", will all go to the first device, and may cause OOM...